### PR TITLE
[time series] Add Time series inputs tests

### PR DESCRIPTION
**Pull request by kashif**
# What does this PR do?
This PR adds tests to make sure that the appropriate inputs are being created for the time series transformer for the training and generation use-cases.

cc @NielsRogge

### CODE REVIEW

File: tests/models/time_series_transformer/test_modeling_time_series_transformer.py [hunk 1]
```diff
@@ -366,6 +367,90 @@ def test_attention_outputs(self):
...
+            "past_observed_mask": torch.arange(history_length, dtype=torch.float32).view(1, history_length),
+        }
+
+        # test with no future_target (only one step prediction)
+        batch["future_time_features"] = torch.arange(history_length, history_length + 1, dtype=torch.float32).view(
+            1, 1, 1
+        )
+        transformer_inputs, loc, scale, _ = model.create_network_inputs(**batch)
+
+        assert (scale == 1.0).all()
```
Comment (NielsRogge) [inline tests/models/time_series_transformer/test_modeling_time_series_transformer.py:411]:
```suggestion
        self.assertTrue((scale == 1.0).all())
```
Please use self.assertTrue everywhere

Comment (sgugger) [reply tests/models/time_series_transformer/test_modeling_time_series_transformer.py:411]: We really don't care about this in test files, both are used in the code base.

**Review by sgugger — APPROVED** [review]
Thanks for cleaning this up!

File: tests/models/time_series_transformer/test_modeling_time_series_transformer.py [hunk 2]
```diff
@@ -366,6 +367,90 @@ def test_attention_outputs(self):
...
+        }
+
+        # test with no future_target (only one step prediction)
+        batch["future_time_features"] = torch.arange(history_length, history_length + 1, dtype=torch.float32).view(
+            1, 1, 1
+        )
+        transformer_inputs, loc, scale, _ = model.create_network_inputs(**batch)
+
+        self.assertTrue((scale == 1.0).all())
+        assert (loc == 0.0).all()
```
Comment (NielsRogge) [inline tests/models/time_series_transformer/test_modeling_time_series_transformer.py:412]: Please update all assert statements, here and below, to use self.assertTrue or another function from unittest

Comment (sgugger) [reply tests/models/time_series_transformer/test_modeling_time_series_transformer.py:412]: I may have not been clear enough in my comment before. WE ABSOLUTELY DON'T CARE AND USE BOTHS IN THE TEST CODEBASE.

Comment (NielsRogge) [reply tests/models/time_series_transformer/test_modeling_time_series_transformer.py:412]: Sorry, I didn't see the previous comment as it was remarked as resolved. Ok for me!

**Pull request merged**